core/commands/swarm.go

```diff
@@ -241,6 +249,8 @@ ipfs swarm connect /ip4/104.131.131.82/tcp/4001/ipfs/QmaCpDMGvV2BGHeYERUEnRQAwe3
 
 		output := make([]string, len(pis))
 		for i, pi := range pis {
+			swrm.Backoff().Clear(pi.ID)
```

Comment:
this is leaking the underlying abstraction. can this be turned into something more "user/client" oriented? Not sure what name would fit